layers/+tools/lsp/README.org

```diff
@@ -55,21 +59,99 @@ A number of configuration variables have been exposed via the LSP layer =config.
 under the derived mode t prefix by =(spacemacs/lsp-bind-keys-for-mode mode)=
 
-| Variable name                   | Default | Description                                                                               |
-|---------------------------------+---------+-------------------------------------------------------------------------------------------|
-| =lsp-navigation=                | `both'  | `simple' or `peek' to bind only xref OR lsp-ui-peek navigation functions                  |
-| =lsp-ui-remap-xref-keybindings= | nil     | When non-nil, xref key bindings remapped to lsp-ui-peek-find-{definition,references}      |
-| =lsp-ui-doc-enable=             | t       | When non-nil, the documentation overlay is displayed                                      |
-| =lsp-ui-doc-include-signature=  | nil     | When nil, signature omitted from lsp-ui-doc overlay (this is usually redundant)           |
-| =lsp-ui-sideline-enable=        | t       | When non-nil, the symbol information overlay is displayed                                 |
-| =lsp-ui-sideline-show-symbol=   | nil     | When non-nil, the symbol information overlay includes symbol name (redundant for c-modes) |
+| Variable name                      | Default | Description                                                                               |
+|------------------------------------+---------+-------------------------------------------------------------------------------------------|
+| =lsp-headerline-breadcrumb-enable= | nil     | When non-nil, shows breadcrumb on headerline.                                             |
+| =lsp-lens-enable=                  | nil     | When non-nil, shows code lens when it's supported.                                        |
+| =lsp-modeline-diagnostics-enable=  | t       | When non-nil, shows error diagnostics in modeline.                                        |
+| =lsp-modeline-code-actions-enable= | t       | When non-nil, shows available code actions in modeline.                                   |
+| =lsp-navigation=                   | `both'  | `simple' or `peek' to bind only xref OR lsp-ui-peek navigation functions                  |
+| =lsp-ui-remap-xref-keybindings=    | nil     | When non-nil, xref key bindings remapped to lsp-ui-peek-find-{definition,references}      |
+| =lsp-ui-doc-enable=                | t       | When non-nil, the documentation overlay is displayed                                      |
+| =lsp-ui-doc-include-signature=     | nil     | When nil, signature omitted from lsp-ui-doc overlay (this is usually redundant)           |
+| =lsp-ui-sideline-enable=           | t       | When non-nil, the symbol information overlay is displayed                                 |
+| =lsp-ui-sideline-show-symbol=      | nil     | When non-nil, the symbol information overlay includes symbol name (redundant for c-modes) |
+
+** Code Lens
+To always display code lens,
```

Comment:
Maybe a short description what code lenses are would be good.

Reply:
> Code lens is a feature that displays ="actionable contextual information interspersed"= in your source code.
In other words, it displays extra information of the source code, and allows you to perform certain actions.
For example, the LSP server may decorate the entry point of a program, and also provides a quick access for running/debugging the program.

How about this?